src/bundle/Resources/public/js/scripts/sidebar/extra.actions.js

```diff
@@ -1,13 +1,13 @@
 (function () {
     const CLASS_HIDDEN = 'ez-extra-actions--hidden';
+    const CLASS_PREVENT_SHOW = 'ez-extra-actions--prevent-show';
     const btns = [...document.querySelectorAll('.ez-btn--extra-actions')];
 
     btns.forEach(btn => {
         btn.addEventListener('click', () => {
             const actions = document.querySelector(`.ez-extra-actions[data-actions="${btn.dataset.actions}"]`);
-            const cssMethodName = actions.classList.contains(CLASS_HIDDEN) ? 'remove' : 'add';
+            const cssMethodName = actions.classList.contains(CLASS_HIDDEN) && !actions.classList.contains(CLASS_PREVENT_SHOW)? 'remove' : 'add';
```

Comment:
Two things: 
1. It's not cssMethodName, because CSS doesn't have methods;
2. To improve readability, I would put `actions.classList.contains(CLASS_HIDDEN) && !actions.classList.contains(CLASS_PREVENT_SHOW)` into some kind of a named variable.